test/integration/scenarios/01.update/verify/manifest_test.go

```diff
@@ -121,6 +121,13 @@ func TestScenario_Update_AllNamespaces(t *testing.T) {
 }
 
 func Scenario_Update(t *testing.T, namespace string) {
+	// Deploy base manifests
```

Comment:
Do we want to reset the manifests between test cases? Otherwise, we are testing accumulated updates across test cases, which I don't think is expected.

Reply:
I would suppose manifests should not be reset to keep the same logic as previous. My change skip applying the manifest in the template folder which is the same as existing symphony objects.